flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -82,7 +83,8 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the {@link CliClient}. */
-public class CliClientTest extends TestLogger {
+@ExtendWith(TestLoggerExtension.class)
```

Comment:
Using the `META-INF/services/org.junit.jupiter.api.extension.Extension` is the better approach here